projects/storefrontlib/src/cms-components/navigation/category-navigation/category-navigation.component.ts

```diff
@@ -21,4 +23,18 @@ export class CategoryNavigationComponent {
     protected componentData: CmsComponentData<CmsNavigationComponent>,
     protected service: NavigationService
   ) {}
+
+  /**
+   * Returns navigation-ui config information.
+   */
+  getNavUIConfig(): Observable<NavigationUiConfig> {
```

Comment:
Usually we pass configurations through the module via `provideConfig()` or `provideDefaultConfig()`. So in this case it would be `category-navigation.module.ts` or whichever imports the component.